### DISCUSSION TITLE
Have a way of not staking the same items from the kits.

### DISCUSSION THREAD

**Discussion by thiagogebrimm** [EssentialsX/Essentials#4239] [Ideas and Feature Suggestions] [answered]
![image](https://user-images.githubusercontent.com/75040244/122136468-f29f5f80-ce18-11eb-8d89-827604957daf.png)

Shulker and totem don't stack, but they end up stacking in kits :(

**Comment by triagonal** [comment, marked as answer]
This behaviour will only occur if the player redeeming the kit has the `essentials.oversizedstacks` permission. If your players don't have this permission, unstackable kit items will not stack. 

You can change the default oversized stack size in the config using the `oversized-stacksize` option.
